lib/collection/src/operations/universal_query/collection_query.rs

```diff
@@ -302,8 +314,8 @@ impl CollectionPrefetch {
             .map(|query| {
                 query.try_into_scoring_query(
                     ids_to_vectors,
-                    lookup_vector_name,
-                    lookup_collection,
+                    parent_lookup_vector_name,
+                    parent_lookup_collection,
```

Comment:
We shouldn't propagate "using" into the prefetches. They should only work on their level